docs/userguide/quickstart.rst

```diff
@@ -22,7 +22,7 @@ be generated with whatever tools that provides a ``build sdist``-alike
 functionality. While this may appear cumbersome, given the added pieces,
 it in fact tremendously enhances the portability of your package. The
 change is driven under `PEP 517 <https://www.python.org/dev/peps/pep-0517/#
-build-requirements>``. To learn more about Python packaging in general,
+build-requirements>`_. To learn more about Python packaging in general,
```

Comment:
Why not use the ```:pep:``` role instead?
```suggestion
change is driven under :pep:`PEP 517 <517#build-requirements>`_.
To learn more about Python packaging in general,
```

Reply:
Sounds good!  GitHub won't let me commit this suggestion for some reason ("Applying suggestions on deleted lines is not supported").

Reply:
Yeah... They have this limitation for some reason. But feel free to apply it manually and add a
```
Co-Authored-By: Sviatoslav Sydorenko <webknjaz@redhat.com>
```
line to the commit message.

Reply:
I have the `[x] Allow edits by maintainers` checkbox checked, so you could push it too ;)

Reply:
Except I'm not a maintainer here. GH doesn't allow to distinguish folks who have write access vs just org members.